xapian-letor/include/xapian-letor/feature.h

```diff
@@ -146,10 +163,10 @@ class XAPIAN_VISIBILITY_DEFAULT Feature {
   private:
 
     /// Don't allow assignment.
-    void operator=(const Feature &);
+    void operator=(const Feature &) = delete;
 
     /// Don't allow copying.
-    Feature(const Feature & o);
+    Feature(const Feature & o) = delete;
```

Comment:
I clearly said not to use C++11 features in API headers.  Just keep these private and don't define them.